-1

**Failed Tests**:   Build 
**Size**: This PR adds an extra 68KB to repository
**Summary**: https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-df1bdf/52965/summary.html
**COMMIT**: f6a0da2ae75a80f1e4563a4b7bd0ae7f052f380a
**CMSSW**: CMSSW_16_1_X_2026-04-29-2300/el8_amd64_gcc13
**User test area**: For local testing, you can use `/cvmfs/cms-ci.cern.ch/week1/cms-sw/cmssw/50831/52965/install.sh` to create a dev area with all the needed externals and cmssw changes.


## Failed Build

I found compilation error when building: 

<pre>>> Compiling  src/SimG4CMS/Calo/src/HcalTestNumberingScheme.cc
/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02939/el8_amd64_gcc13/external/gcc/13.4.0-6908cfdf803923e783448096ca4f0923/bin/c++ -c -DCMS_MICRO_ARCH='x86-64-v3' -DGNU_GCC -D_GNU_SOURCE -DTBB_USE_GLIBCXX_VERSION=130400 -DTBB_SUPPRESS_DEPRECATED_MESSAGES -DTBB_PREVIEW_RESUMABLE_TASKS=1 -DTBB_PREVIEW_TASK_GROUP_EXTENSIONS=1 -DBOOST_SPIRIT_THREADSAFE -DPHOENIX_THREADSAFE -DBOOST_MATH_DISABLE_STD_FPCLASSIFY -DBOOST_UUID_RANDOM_PROVIDER_FORCE_POSIX -DBOOST_MPL_IGNORE_PARENTHESES_WARNING -DGNU_GCC -DG4V9 -DGNU_GCC -DG4V9 -DDD4HEP_USE_GEANT4_UNITS=1 -DCMSSW_GIT_HASH='CMSSW_16_1_X_2026-04-29-2300' -DPROJECT_NAME='CMSSW' -DPROJECT_VERSION='CMSSW_16_1_X_2026-04-29-2300' -Isrc -Ipoison -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02939/el8_amd64_gcc13/cms/cmssw-patch/CMSSW_16_1_X_2026-04-29-2300/src -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02939/el8_amd64_gcc13/external/dd4hep/v01-31-0x-850311dcca6e15221c080eae3e6e846f/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02939/el8_amd64_gcc13/external/geant4/11.2.2-28be4c4ed0c6044511df62789f5f78bd/include/Geant4 -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02939/el8_amd64_gcc13/external/geant4/11.2.2-28be4c4ed0c6044511df62789f5f78bd/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02939/el8_amd64_gcc13/external/pcre/8.43-6d98fda3bfd074ebb583e2d6a2c75d25/include -isystem/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02939/el8_amd64_gcc13/external/boost/1.80.0-6429d45cb860600db5fc5d037104a1f8/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02939/el8_amd64_gcc13/external/bz2lib/1.0.6-d113e1c6278c07eeaff5f84db9548446/include -isystem/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02939/el8_amd64_gcc13/external/clhep/2.4.7.2-a1886bcfe0fba01667305d6d8ed03de5/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02939/el8_amd64_gcc13/external/expat/2.4.8-90d4121d63d495f95df72519ccbb6e9c/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02939/el8_amd64_gcc13/external/gsl/2.6-9011a41928244b609ca4c22c439b3fef/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02939/el8_amd64_gcc13/external/hepmc/2.06.10-903e23142392c852f3df97c32fbd4659/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02939/el8_amd64_gcc13/external/libuuid/2.34-5ba7a8abfc0c5fecdc448cca360c25ff/include -isystem/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02939/el8_amd64_gcc13/lcg/root/6.36.11-c9684ba7622be6ba003c6d1250a79c91/include -isystem/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02939/el8_amd64_gcc13/external/tbb/v2022.3.0-88eb7be4ee320d604a798a914aea6359/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02939/el8_amd64_gcc13/external/vecgeom/v1.2.11-490889b9b72917b47f9aa6764895a8c7/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02939/el8_amd64_gcc13/external/vecgeom/v1.2.11-490889b9b72917b47f9aa6764895a8c7/include/VecGeom -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02939/el8_amd64_gcc13/external/xerces-c/3.1.3-ab719034dc2061fa61b09353203a0c2d/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02939/el8_amd64_gcc13/external/xz/5.6.4-b9c4ffbc390ed320a5d57fd552e29a05/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02939/el8_amd64_gcc13/external/zlib/1.2.13-589f6bb51bbeba38a7adf5a10ea8a093/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02939/el8_amd64_gcc13/external/alpaka/2.1.1-3caaac8d71f39d400ab2511b2403675a/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02939/el8_amd64_gcc13/external/eigen/3bb6a48d8c171cf20b5f8e48bfb4e424fbd4f79e-95c02b8a883b2934decb8bb53ff9b486/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02939/el8_amd64_gcc13/external/eigen/3bb6a48d8c171cf20b5f8e48bfb4e424fbd4f79e-95c02b8a883b2934decb8bb53ff9b486/include/eigen3 -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02939/el8_amd64_gcc13/external/fmt/10.2.1-31d67b0504b4ba2262f03d3c5cad83c1/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02939/el8_amd64_gcc13/external/md5/1.0.0-26057075013e190e56dad37d35219376/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02939/el8_amd64_gcc13/external/OpenBLAS/0.3.27-da4a3c2bb8ae43f3913a4a44acdb1b50/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02939/el8_amd64_gcc13/external/tinyxml2/6.2.0-67924ead96ecb4e69aad321b767979a5/include -O3 -pthread -pipe -Werror=main -Werror=pointer-arith -Werror=overlength-strings -Wno-vla -Werror=overflow -std=c++20 -ftree-vectorize -Werror=array-bounds -Werror=format-contains-nul -Werror=type-limits -fvisibility-inlines-hidden -fno-math-errno --param vect-max-version-for-alias-checks=50 -Xassembler --compress-debug-sections -Wno-error=array-bounds -Warray-bounds -fuse-ld=bfd -march=x86-64-v3 -felide-constructors -fmessage-length=0 -Wall -Wno-non-template-friend -Wno-long-long -Wreturn-type -Wextra -Wpessimizing-move -Wclass-memaccess -Wno-cast-function-type -Wno-unused-but-set-parameter -Wno-ignored-qualifiers -Wno-unused-parameter -Wunused -Wparentheses -Werror=return-type -Werror=unused-value -Werror=unused-label -Werror=address -Werror=format -Werror=sign-compare -Werror=write-strings -Werror=delete-non-virtual-dtor -Werror=strict-aliasing -Werror=narrowing -Werror=unused-but-set-variable -Werror=reorder -Werror=unused-variable -Werror=conversion-null -Werror=return-local-addr -Wnon-virtual-dtor -Werror=switch -fdiagnostics-show-option -Wno-unused-local-typedefs -Wno-attributes -Wno-psabi -DEIGEN_DONT_PARALLELIZE -DEIGEN_MAX_ALIGN_BYTES=64 -DALPAKA_DEFAULT_HOST_MEMORY_ALIGNMENT=128 -DALPAKA_DISABLE_VENDOR_RNG -DALPAKA_HAS_STD_ATOMIC_REF -Wno-error=unused-variable -ftls-model=global-dynamic -pthread -DBOOST_DISABLE_ASSERTS -flto=auto -fipa-icf -flto-odr-type-merging -fno-fat-lto-objects -Wodr -fPIC -MMD -MF tmp/el8_amd64_gcc13/src/SimG4CMS/Calo/src/SimG4CMSCalo/HcalTestNumberingScheme.cc.d src/SimG4CMS/Calo/src/HcalTestNumberingScheme.cc -o tmp/el8_amd64_gcc13/src/SimG4CMS/Calo/src/SimG4CMSCalo/HcalTestNumberingScheme.cc.o
>> Compiling bigobj src/SimG4CMS/Calo/src/CaloDetInfo.cc
/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02939/el8_amd64_gcc13/external/gcc/13.4.0-6908cfdf803923e783448096ca4f0923/bin/c++ -c -DCMS_MICRO_ARCH='x86-64-v3' -DGNU_GCC -D_GNU_SOURCE -DTBB_USE_GLIBCXX_VERSION=130400 -DTBB_SUPPRESS_DEPRECATED_MESSAGES -DTBB_PREVIEW_RESUMABLE_TASKS=1 -DTBB_PREVIEW_TASK_GROUP_EXTENSIONS=1 -DBOOST_SPIRIT_THREADSAFE -DPHOENIX_THREADSAFE -DBOOST_MATH_DISABLE_STD_FPCLASSIFY -DBOOST_UUID_RANDOM_PROVIDER_FORCE_POSIX -DBOOST_MPL_IGNORE_PARENTHESES_WARNING -DGNU_GCC -DG4V9 -DGNU_GCC -DG4V9 -DDD4HEP_USE_GEANT4_UNITS=1 -DCMSSW_GIT_HASH='CMSSW_16_1_X_2026-04-29-2300' -DPROJECT_NAME='CMSSW' -DPROJECT_VERSION='CMSSW_16_1_X_2026-04-29-2300' -Isrc -Ipoison -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02939/el8_amd64_gcc13/cms/cmssw-patch/CMSSW_16_1_X_2026-04-29-2300/src -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02939/el8_amd64_gcc13/external/dd4hep/v01-31-0x-850311dcca6e15221c080eae3e6e846f/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02939/el8_amd64_gcc13/external/geant4/11.2.2-28be4c4ed0c6044511df62789f5f78bd/include/Geant4 -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02939/el8_amd64_gcc13/external/geant4/11.2.2-28be4c4ed0c6044511df62789f5f78bd/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02939/el8_amd64_gcc13/external/pcre/8.43-6d98fda3bfd074ebb583e2d6a2c75d25/include -isystem/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02939/el8_amd64_gcc13/external/boost/1.80.0-6429d45cb860600db5fc5d037104a1f8/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02939/el8_amd64_gcc13/external/bz2lib/1.0.6-d113e1c6278c07eeaff5f84db9548446/include -isystem/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02939/el8_amd64_gcc13/external/clhep/2.4.7.2-a1886bcfe0fba01667305d6d8ed03de5/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02939/el8_amd64_gcc13/external/expat/2.4.8-90d4121d63d495f95df72519ccbb6e9c/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02939/el8_amd64_gcc13/external/gsl/2.6-9011a41928244b609ca4c22c439b3fef/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02939/el8_amd64_gcc13/external/hepmc/2.06.10-903e23142392c852f3df97c32fbd4659/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02939/el8_amd64_gcc13/external/libuuid/2.34-5ba7a8abfc0c5fecdc448cca360c25ff/include -isystem/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02939/el8_amd64_gcc13/lcg/root/6.36.11-c9684ba7622be6ba003c6d1250a79c91/include -isystem/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02939/el8_amd64_gcc13/external/tbb/v2022.3.0-88eb7be4ee320d604a798a914aea6359/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02939/el8_amd64_gcc13/external/vecgeom/v1.2.11-490889b9b72917b47f9aa6764895a8c7/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02939/el8_amd64_gcc13/external/vecgeom/v1.2.11-490889b9b72917b47f9aa6764895a8c7/include/VecGeom -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02939/el8_amd64_gcc13/external/xerces-c/3.1.3-ab719034dc2061fa61b09353203a0c2d/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02939/el8_amd64_gcc13/external/xz/5.6.4-b9c4ffbc390ed320a5d57fd552e29a05/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02939/el8_amd64_gcc13/external/zlib/1.2.13-589f6bb51bbeba38a7adf5a10ea8a093/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02939/el8_amd64_gcc13/external/alpaka/2.1.1-3caaac8d71f39d400ab2511b2403675a/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02939/el8_amd64_gcc13/external/eigen/3bb6a48d8c171cf20b5f8e48bfb4e424fbd4f79e-95c02b8a883b2934decb8bb53ff9b486/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02939/el8_amd64_gcc13/external/eigen/3bb6a48d8c171cf20b5f8e48bfb4e424fbd4f79e-95c02b8a883b2934decb8bb53ff9b486/include/eigen3 -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02939/el8_amd64_gcc13/external/fmt/10.2.1-31d67b0504b4ba2262f03d3c5cad83c1/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02939/el8_amd64_gcc13/external/md5/1.0.0-26057075013e190e56dad37d35219376/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02939/el8_amd64_gcc13/external/OpenBLAS/0.3.27-da4a3c2bb8ae43f3913a4a44acdb1b50/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02939/el8_amd64_gcc13/external/tinyxml2/6.2.0-67924ead96ecb4e69aad321b767979a5/include -O3 -pthread -pipe -Werror=main -Werror=pointer-arith -Werror=overlength-strings -Wno-vla -Werror=overflow -std=c++20 -ftree-vectorize -Werror=array-bounds -Werror=format-contains-nul -Werror=type-limits -fvisibility-inlines-hidden -fno-math-errno --param vect-max-version-for-alias-checks=50 -Xassembler --compress-debug-sections -Wno-error=array-bounds -Warray-bounds -fuse-ld=bfd -march=x86-64-v3 -felide-constructors -fmessage-length=0 -Wall -Wno-non-template-friend -Wno-long-long -Wreturn-type -Wextra -Wpessimizing-move -Wclass-memaccess -Wno-cast-function-type -Wno-unused-but-set-parameter -Wno-ignored-qualifiers -Wno-unused-parameter -Wunused -Wparentheses -Werror=return-type -Werror=unused-value -Werror=unused-label -Werror=address -Werror=format -Werror=sign-compare -Werror=write-strings -Werror=delete-non-virtual-dtor -Werror=strict-aliasing -Werror=narrowing -Werror=unused-but-set-variable -Werror=reorder -Werror=conversion-null -Werror=return-local-addr -Wnon-virtual-dtor -Werror=switch -fdiagnostics-show-option -Wno-unused-local-typedefs -Wno-attributes -Wno-psabi -DEIGEN_DONT_PARALLELIZE -DEIGEN_MAX_ALIGN_BYTES=64 -DALPAKA_DEFAULT_HOST_MEMORY_ALIGNMENT=128 -DALPAKA_DISABLE_VENDOR_RNG -DALPAKA_HAS_STD_ATOMIC_REF -Wno-error=unused-variable -ftls-model=global-dynamic -pthread -DBOOST_DISABLE_ASSERTS -flto=auto -fipa-icf -flto-odr-type-merging -fno-fat-lto-objects -Wodr -Wl,--exclude-libs,ALL -fPIC src/SimG4CMS/Calo/src/CaloDetInfo.cc -o tmp/el8_amd64_gcc13/src/SimG4CMS/Calo/src/SimG4CMSCalo/bigobj/CaloDetInfo.cc.o
src/SimG4CMS/Calo/src/HGCalNumberingScheme.cc: In member function 'uint32_t HGCalNumberingScheme::getUnitID(int, int, int, int, const G4ThreeVector&, double&)':
src/SimG4CMS/Calo/src/HGCalNumberingScheme.cc:192:81: error: 'const class HGCalDDDConstants' has no member named 'tileSiPM'
  192 |                                    << " SiPM " << typm.second << ":" << hgcons_.tileSiPM(typm.second) << " Layer "
      |                                                                                 ^~~~~~~~
>> Compiling bigobj src/SimG4CMS/Calo/src/CaloG4Hit.cc
/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02939/el8_amd64_gcc13/external/gcc/13.4.0-6908cfdf803923e783448096ca4f0923/bin/c++ -c -DCMS_MICRO_ARCH='x86-64-v3' -DGNU_GCC -D_GNU_SOURCE -DTBB_USE_GLIBCXX_VERSION=130400 -DTBB_SUPPRESS_DEPRECATED_MESSAGES -DTBB_PREVIEW_RESUMABLE_TASKS=1 -DTBB_PREVIEW_TASK_GROUP_EXTENSIONS=1 -DBOOST_SPIRIT_THREADSAFE -DPHOENIX_THREADSAFE -DBOOST_MATH_DISABLE_STD_FPCLASSIFY -DBOOST_UUID_RANDOM_PROVIDER_FORCE_POSIX -DBOOST_MPL_IGNORE_PARENTHESES_WARNING -DGNU_GCC -DG4V9 -DGNU_GCC -DG4V9 -DDD4HEP_USE_GEANT4_UNITS=1 -DCMSSW_GIT_HASH='CMSSW_16_1_X_2026-04-29-2300' -DPROJECT_NAME='CMSSW' -DPROJECT_VERSION='CMSSW_16_1_X_2026-04-29-2300' -Isrc -Ipoison -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02939/el8_amd64_gcc13/cms/cmssw-patch/CMSSW_16_1_X_2026-04-29-2300/src -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02939/el8_amd64_gcc13/external/dd4hep/v01-31-0x-850311dcca6e15221c080eae3e6e846f/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02939/el8_amd64_gcc13/external/geant4/11.2.2-28be4c4ed0c6044511df62789f5f78bd/include/Geant4 -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02939/el8_amd64_gcc13/external/geant4/11.2.2-28be4c4ed0c6044511df62789f5f78bd/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02939/el8_amd64_gcc13/external/pcre/8.43-6d98fda3bfd074ebb583e2d6a2c75d25/include -isystem/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02939/el8_amd64_gcc13/external/boost/1.80.0-6429d45cb860600db5fc5d037104a1f8/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02939/el8_amd64_gcc13/external/bz2lib/1.0.6-d113e1c6278c07eeaff5f84db9548446/include -isystem/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02939/el8_amd64_gcc13/external/clhep/2.4.7.2-a1886bcfe0fba01667305d6d8ed03de5/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02939/el8_amd64_gcc13/external/expat/2.4.8-90d4121d63d495f95df72519ccbb6e9c/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02939/el8_amd64_gcc13/external/gsl/2.6-9011a41928244b609ca4c22c439b3fef/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02939/el8_amd64_gcc13/external/hepmc/2.06.10-903e23142392c852f3df97c32fbd4659/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02939/el8_amd64_gcc13/external/libuuid/2.34-5ba7a8abfc0c5fecdc448cca360c25ff/include -isystem/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02939/el8_amd64_gcc13/lcg/root/6.36.11-c9684ba7622be6ba003c6d1250a79c91/include -isystem/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02939/el8_amd64_gcc13/external/tbb/v2022.3.0-88eb7be4ee320d604a798a914aea6359/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02939/el8_amd64_gcc13/external/vecgeom/v1.2.11-490889b9b72917b47f9aa6764895a8c7/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02939/el8_amd64_gcc13/external/vecgeom/v1.2.11-490889b9b72917b47f9aa6764895a8c7/include/VecGeom -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02939/el8_amd64_gcc13/external/xerces-c/3.1.3-ab719034dc2061fa61b09353203a0c2d/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02939/el8_amd64_gcc13/external/xz/5.6.4-b9c4ffbc390ed320a5d57fd552e29a05/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02939/el8_amd64_gcc13/external/zlib/1.2.13-589f6bb51bbeba38a7adf5a10ea8a093/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02939/el8_amd64_gcc13/external/alpaka/2.1.1-3caaac8d71f39d400ab2511b2403675a/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02939/el8_amd64_gcc13/external/eigen/3bb6a48d8c171cf20b5f8e48bfb4e424fbd4f79e-95c02b8a883b2934decb8bb53ff9b486/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02939/el8_amd64_gcc13/external/eigen/3bb6a48d8c171cf20b5f8e48bfb4e424fbd4f79e-95c02b8a883b2934decb8bb53ff9b486/include/eigen3 -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02939/el8_amd64_gcc13/external/fmt/10.2.1-31d67b0504b4ba2262f03d3c5cad83c1/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02939/el8_amd64_gcc13/external/md5/1.0.0-26057075013e190e56dad37d35219376/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02939/el8_amd64_gcc13/external/OpenBLAS/0.3.27-da4a3c2bb8ae43f3913a4a44acdb1b50/include -I/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02939/el8_amd64_gcc13/external/tinyxml2/6.2.0-67924ead96ecb4e69aad321b767979a5/include -O3 -pthread -pipe -Werror=main -Werror=pointer-arith -Werror=overlength-strings -Wno-vla -Werror=overflow -std=c++20 -ftree-vectorize -Werror=array-bounds -Werror=format-contains-nul -Werror=type-limits -fvisibility-inlines-hidden -fno-math-errno --param vect-max-version-for-alias-checks=50 -Xassembler --compress-debug-sections -Wno-error=array-bounds -Warray-bounds -fuse-ld=bfd -march=x86-64-v3 -felide-constructors -fmessage-length=0 -Wall -Wno-non-template-friend -Wno-long-long -Wreturn-type -Wextra -Wpessimizing-move -Wclass-memaccess -Wno-cast-function-type -Wno-unused-but-set-parameter -Wno-ignored-qualifiers -Wno-unused-parameter -Wunused -Wparentheses -Werror=return-type -Werror=unused-value -Werror=unused-label -Werror=address -Werror=format -Werror=sign-compare -Werror=write-strings -Werror=delete-non-virtual-dtor -Werror=strict-aliasing -Werror=narrowing -Werror=unused-but-set-variable -Werror=reorder -Werror=conversion-null -Werror=return-local-addr -Wnon-virtual-dtor -Werror=switch -fdiagnostics-show-option -Wno-unused-local-typedefs -Wno-attributes -Wno-psabi -DEIGEN_DONT_PARALLELIZE -DEIGEN_MAX_ALIGN_BYTES=64 -DALPAKA_DEFAULT_HOST_MEMORY_ALIGNMENT=128 -DALPAKA_DISABLE_VENDOR_RNG -DALPAKA_HAS_STD_ATOMIC_REF -Wno-error=unused-variable -ftls-model=global-dynamic -pthread -DBOOST_DISABLE_ASSERTS -flto=auto -fipa-icf -flto-odr-type-merging -fno-fat-lto-objects -Wodr -Wl,--exclude-libs,ALL -fPIC src/SimG4CMS/Calo/src/CaloG4Hit.cc -o tmp/el8_amd64_gcc13/src/SimG4CMS/Calo/src/SimG4CMSCalo/bigobj/CaloG4Hit.cc.o
>> Compiling bigobj src/SimG4CMS/Calo/src/CaloGVHit.cc
</pre>

--- a/SimG4CMS/Calo/src/HGCalNumberingScheme.cc
+++ b/SimG4CMS/Calo/src/HGCalNumberingScheme.cc
@@ -189,9 +189,8 @@ uint32_t HGCalNumberingScheme::getUnitID(int layer, int module, int cell, int iz
       }
       if (debug)
         edm::LogVerbatim("HGCSim") << "Radius/Phi " << id[0] << ":" << id[1] << " Type " << id[2] << ":" << typm.first
-                                   << " SiPM " << typm.second << ":" << hgcons_.tileSiPM(typm.second) << " Layer "
-                                   << layer << " z " << iz << " ID " << detId << " wt " << wt << " position " << pos
-                                   << " R " << pos.perp();
+                                   << " SiPM " << typm.second << " Layer " << layer << " z " << iz << " ID " << detId
+                                   << " wt " << wt << " position " << pos << " R " << pos.perp();
 #ifdef EDM_ML_DEBUG
     } else {
       edm::LogVerbatim("HGCSim") << "Radius/Phi " << id[0] << ":" << id[1] << " Type " << id[2] << " Layer|iz " << layer

